src/components/ConditionalFilter/API/OrderFilterAPIProvider.tsx

```diff
@@ -81,7 +81,7 @@ const createAPIHandler = (
   if (rowType === "customer") {
     return new TextInputValuesHandler([
```

Comment:
so if this is not email only should it not be email type? I should not be able to enter something that is not the email